File: Entities/Admission.cs
namespace PrivateHospitalSystem.Entities
{
    public class Admission
    {
        public Guid Id { get; set; }

        public Guid PatientId { get; set; }
        public Patient? Patient { get; set; }

        public Guid BedId { get; set; }
        public Bed? Bed { get; set; }

        public DateTime AdmittedAt { get; set; } = DateTime.UtcNow;
        public DateTime? DischargedAt { get; set; } 

        public string? Reason { get; set; } 

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: frontend/src/types/admission.ts
export interface Admission {
  id: string
  patientId: string
  patientName: string
  patientNumber?: string
  bedId: string
  bedNumber: string
  department?: string
  roomNumber?: string
  admittedAt: string
  admissionDate?: string
  dischargedAt: string | null
  reason: string | null
}

export interface CreateAdmissionDto {
  patientId: string
  bedId: string
  reason?: string
}